### DISCUSSION TITLE
v2.2.0

### DISCUSSION THREAD

**Discussion by imdevarsh** [imdevarsh/meowclient#20] [Announcements]
## What's Changed
* Make CloudConnection an EventEmitter by @hello-smile6 in https://github.com/webdev03/meowclient/pull/19

## New Contributors
* @hello-smile6 made their first contribution in https://github.com/webdev03/meowclient/pull/19

**Full Changelog**: https://github.com/webdev03/meowclient/compare/v2.1.0...v2.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/webdev03/meowclient/releases/tag/v2.2.0'>v2.2.0</a>.</em>

**Comment by hello-smile6** [comment]
Was that one PR really worth a new version?

**Reply by imdevarsh** [reply]
Didn't have much planned

**Reply by hello-smile6** [reply]
Oh.
